src/index.js

```diff
@@ -1,39 +1,237 @@
-import './style.css';
-
-const lists = [
-  {
-    index: 2,
-    description: 'PHP',
-    completed: true,
-  },
-  {
-    index: 0,
-    description: 'JS',
-    completed: true,
-  },
-  {
-    index: 1,
-    description: 'SWAHILI',
-    completed: true,
-  },
-  {
-    index: 3,
-    description: 'rwanda',
-    completed: true,
-  },
-];
-
-const renderlists = () => {
-  const listselector = document.querySelector('.list-detail');
-  let render = '';
-  lists.sort((x, y) => x.index - y.index).forEach((listItem) => {
-    render += ` 
-      <li>
-        <input type="checkbox" class="checkbox" ${lists.completed ? 'checked' : ''} > ${listItem.description}
-      </li>
-      `;
-  });
-  listselector.innerHTML = render;
+/* eslint-disable */
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.